bindings/c/local.mk

```diff
@@ -26,6 +26,10 @@ fdb_c_LIBS := lib/libfdbclient.a lib/libfdbrpc.a lib/libflow.a
 fdb_c_tests_LIBS := -Llib -lfdb_c
 fdb_c_tests_HEADERS := -Ibindings/c
 
+ifndef __TLS_DISABLED__
+fdb_c_LIBS += lib/libFDBLibTLS.a /usr/local/lib/libtls.a /usr/local/lib/libssl.a /usr/local/lib/libcrypto.a
```

Comment:
I think it'd be better do the same `$(CC) $(CFLAGS) -print-file-name libtls.a` sort of trick for each static library, and assign that to some ${TLS_LIBRARIES} variable, so that we don't have to pin the libraries to a specific file path.

And having a TLS_LIBRARIES variable that holds the above list of libraries if it is enabled and nothing if it's disabled lets you remove all the other `ifndef __TLS_DISABLED__`'s from the Makefile.

Was there a reason you decided not to go the `-Bstatic -l tls -l ssl -l crypto -Bdynamic` route, as that'd make the above explicit file path naming unnecessary.